### PR TITLE
project init

### PR DESCRIPTION
**Pull request by benyl-nie**

### CODE REVIEW

**Comment by jovany-wang** [comment]
@senyer cc

File: dst-sites/src/component/docker.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+  render() {
+    return (
+      <React.Fragment></React.Fragment>
+    );
+  }
+}
```
Comment (senyer) [inline dst-sites/src/component/docker.jsx]: blank line

File: dst-sites/src/component/hub.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+  render() {
+    return (
+      <React.Fragment></React.Fragment>
+    );
+  }
+}
```
Comment (senyer) [inline dst-sites/src/component/hub.jsx]: blank line

File: dst-sites/src/component/difference.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+  render() {
+    return (
+      <React.Fragment></React.Fragment>
+    );
+  }
+}
```
Comment (senyer) [inline dst-sites/src/component/difference.jsx]: blank line

File: dst-sites/src/component/contract.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  constructor(props){
+    super(props);
+    this.state = {}
+  }
+  render() {
+    return (
+      <React.Fragment></React.Fragment>
+    );
+  }
+}
```
Comment (senyer) [inline dst-sites/src/component/contract.jsx]: blank line

File: dst-sites/src/component/introduce.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  render() {
+    return (
+      <div className='introduce'>
+        <div className='container'>
+          <div className='header'>
+            <div className='logo'>
+              <span>docker</span>
+            </div>
+            <div className='menu'>
+              <span>Why docker?</span>
```
Comment (senyer) [inline dst-sites/src/component/introduce.jsx]: no docker

**Comment by senyer** [comment]
Well Done

**Comment by senyer** [comment]
btw,I think you should remove useless comment in your code

File: dst-sites/src/component/docker.jsx (hunk 1 above)
Comment (benyl-nie) [reply dst-sites/src/component/docker.jsx]: done

File: dst-sites/src/component/hub.jsx (hunk 2 above)
Comment (benyl-nie) [reply dst-sites/src/component/hub.jsx]: done

File: dst-sites/src/component/difference.jsx (hunk 3 above)
Comment (benyl-nie) [reply dst-sites/src/component/difference.jsx]: done

File: dst-sites/src/component/contract.jsx (hunk 4 above)
Comment (benyl-nie) [reply dst-sites/src/component/contract.jsx]: done

File: dst-sites/src/component/introduce.jsx (hunk 5 above)
Comment (benyl-nie) [reply dst-sites/src/component/introduce.jsx]: 待文案修改后 修改整体

**Pull request merged**